.gitignore

```diff
@@ -1,39 +1,52 @@
-*.pyc
```

Comment:
why has this file changed?

Reply:
I organized it a bit and added IDE files (*.vscode* if I remember correctly).